etc/config/wazuh-agent.yml

```diff
@@ -24,3 +24,9 @@ logcollector:
     - /var/log/auth.log
   reload_interval: 1m
   file_wait: 500ms
+  channel_refresh: 5s
+  windows:
+    - channel: Application
```

Comment:
These are just examples, right?

Reply:
Yes, as the `- /var/log/auth.log` in localfiles but it can be taken out of the file

Reply:
Since this is the default configuration in a clean installation, maybe this can be removed.